src/home/homework-luks.c

```diff
@@ -1950,6 +1955,8 @@ static int calculate_initial_image_size(UserRecord *h, int image_fd, const char
         }
 
         lower_boundary = minimal_size_by_fs_name(fstype);
+        if (lower_boundary != UINT64_MAX)
+                lower_boundary += GPT_LUKS2_OVERHEAD;
```

Comment:
overflow check?

Reply:
wel, both are constant "lower" boundaries if you so will, if they'd overflow we'd be in deep trouble... ;-)

But yeah, makes sense to encode that fact in an assert()